main.tf

```diff
@@ -64,5 +64,6 @@ resource "aws_acmpca_certificate_authority" "this" {
   }
 }
 
-data "aws_caller_identity" "current" {}
-
+output "CSR" {
```

Comment:
Outputs should be in a separate file, `outputs.tf`. Also, rename this to spell it out, `certificate_signing_request`